bootstrap/configs.go

```diff
@@ -17,7 +17,7 @@ import (
 // MGChannels is a list of Magistrala Channels corresponding Magistrala Thing connects to.
 type Config struct {
 	ThingID     string    `json:"thing_id"`
-	Owner       string    `json:"owner,omitempty"`
+	Owner       string    `json:"domain_id,omitempty"`
```

Comment:
```suggestion
	DomainID       string    `json:"domain_id,omitempty"`
```

Replace this everywhere